NDTensors/src/lib/CUDAExtensions/src/CUDAExtensions.jl

```diff
@@ -0,0 +1,5 @@
+module CUDAExtensions
+include("cuda.jl")
+
+export cu
```

Comment:
So that would mean externally to use this function you would now need to write `NDTensors.CUDAExtensions.cu` is that okay?

Reply:
If the concern is that it is a breaking change since users have been using `NDTensors.cu`, we can put `using .CUDAExtensions: cu` somewhere in the `NDTensors` module to still allow that syntax.

Reply:
I believe this is resolved, we added `using NDTensors.CUDAExtensions.cu` to `imports.jl`